src/pages/home/index.js

```diff
@@ -66,15 +67,24 @@ const Home = () => {
     setTabSelected(dataFromChild);
   };
 
+  const title = tabSelected => {
+    if (tabSelected === CREATE) {
+      return 'sideBar.create.title';
+    } else if (tabSelected === EDIT) {
+      return 'profile.edit.subtitle';
+    } else {
+      return 'menu.chats';
+    }
```

Comment:
Sometimes `else if` are quite messy. You could do something like this, check if it works.
```suggestion
    if (tabSelected === CREATE) {
      return 'sideBar.create.title';
    } 
    if (tabSelected === EDIT) {
      return 'profile.edit.subtitle';
    }
    return 'menu.chats';
```
It's much more readble in my opinion. But once again, this is just an opinion!